services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.dateparse import parse_date
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
+                         movie_id: id,
+                         cinema_hall_id: int) -> Movie | CinemaHall:
+    movie=Movie.objects.get(id=movie_id)
+    cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+    return Movie.objects.create(
+        movie_show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: str) -> list[Movie | CinemaHall]:
```

Comment:
The return type hint for `get_movies_sessions` is incorrect. It should be `QuerySet[MovieSession]` instead of `list[Movie | CinemaHall]`.